packages/trader/src/App/Components/Elements/VideoStream/volume-control.tsx

```diff
@@ -0,0 +1,177 @@
+
+        is_dragging.current = true;
+    };
+
+    const onRewind = (e: React.MouseEvent<HTMLElement, MouseEvent>) => {
+        e.preventDefault();
+        e.stopPropagation();
+
+        if ((e.target as HTMLElement).className === 'player__volume-dot') return;
+        if (!volume_bar_ref?.current) return;
+
+        const new_height = calculateNewHight(e);
+
+        volume_bar_ref.current.style.setProperty('transition', 'none');
+        volume_bar_ref.current.style.setProperty('height', `${new_height}%`);
+
+        onVolumeChange(new_height / 100);
+        checkVolumeControl(new_height);
+    };
+
+    React.useEffect(() => {
+        document.addEventListener('mousemove', mouseMoveHandler as unknown as (this: Document, ev: MouseEvent) => void);
+        document.addEventListener('mouseup', mouseUpHandler as unknown as (this: Document, ev: MouseEvent) => void);
+
+        return () => {
+            document.removeEventListener(
+                'mousemove',
+                mouseMoveHandler as unknown as (this: Document, ev: MouseEvent) => void
+            );
+            document.removeEventListener(
+                'mouseup',
+                mouseUpHandler as unknown as (this: Document, ev: MouseEvent) => void
+            );
+        };
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
+
+    return (
+        <div
+            className='player__volume__wrapper'
+            onMouseOver={is_mobile ? undefined : () => setShowVolume(true)}
+            onMouseLeave={is_mobile || is_dragging.current ? undefined : () => setShowVolume(false)}
+        >
+            <button className='player__controls__button' onClick={buttonClickHandler}>
+                <Icon
+                    icon={is_muted ? 'IcSoundOff' : 'IcSoundOn'}
+                    custom_color='var(--text-colored-background)'
+                    height={18}
+                    width={20}
+                />
+            </button>
+            <CSSTransition
+                in={show_volume}
+                timeout={100}
+                classNames={{
+                    enter: 'player__volume-bar__wrapper--enter',
+                    enterDone: 'player__volume-bar__wrapper--enter-done',
+                    exit: 'player__volume-bar__wrapper--exit',
+                }}
+                unmountOnExit
+            >
+                <div className='player__volume-bar__wrapper'>
+                    <div className='player__volume-bar' onClick={e => onRewind(e)} onKeyDown={undefined}>
+                        <div
+                            className='player__volume-bar__filled'
+                            ref={volume_bar_ref}
+                            style={{ height: `${is_muted ? 0 : (volume ?? 0.5) * 100}%` }}
+                        >
+                            <span
+                                className='player__volume-dot'
+                                onMouseDown={e => mouseDownHandler(e)}
```

Comment:
```suggestion
                                onMouseDown={mouseDownHandler}
```
same for all other callback that only pass event to another callback :)